installing/installing_gcp/preparing-to-install-on-gcp.adoc

```diff
@@ -0,0 +1,46 @@
+
+* **xref:../../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[Installing a cluster on GCP with network customizations]**: You can customize your {product-title} network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements.
+
+* **xref:../../installing/installing_gcp/installing-restricted-networks-gcp.adoc#installing-restricted-networks-gcp[Installing a cluster on GCP by using an internal mirror]**: You can install {product-title} on GCP infrastructure that you provide by using an internal mirror of the installation release content. You can use this method to install a cluster that does not require an active Internet connection to obtain the software components. While you can install {product-title} by using the mirrored content, your cluster still requires Internet access to use the GCP APIs.
+
+* **xref:../../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[Installing a cluster on an existing Virtual Private Cloud]**: You can install {product-title} on an existing GCP Virtual Private Cloud (VPC). You can use this installation method if you have constraints set by the guidelines of your company, such as limits when creating new accounts or infrastructure.
+
+* **xref:../../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[Installing a private cluster on an existing VPC]**: You can install a private cluster on an existing GCP VPC. You can use this method to deploy {product-title} on an internal network that is not visible to the Internet.
+
+* **xref:../../installing/installing_gcp/installing-gcp-user-infra.adoc#installing-gcp-user-infra[Installing a cluster on GCP infrastructure that you provide]**: You can install {product-title} on GCP infrastructure that you provide. You can use the provided CloudFormation templates to create stacks of AWS resources that represent each of the components required for an {product-title} installation.
+
+* **xref:../../installing/installing_gcp/installing-restricted-networks-gcp.adoc#installing-restricted-networks-gcp[Installing a cluster on GCP in a restricted network with user-provisioned ifrastructure]**: You can install {product-title} on GCP in a restricted network with user-provisioned infrastructure. You can use the provided templates to create GCP resources that represent restricted networks with user-provisioned infrastructure.
+
+[id="preparing-to-install-on-gcp-next-steps"]
+== Next steps
+
+* xref:../../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Installing a GCP account]
```

Comment:
You could update this link to mirror the title of the assembly that it links to, which is "Configuring a GCP project". I'm not very familiar with GCP, but a "GCP project" seems specific.